etebase_server/settings.py

```diff
@@ -126,13 +126,12 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/3.0/howto/static-files/
 
+STATIC_ROOT = os.path.join(BASE_DIR, "static")
 STATIC_URL = "/static/"
-STATIC_ROOT = os.environ.get("DJANGO_STATIC_ROOT", os.path.join(BASE_DIR, "static"))
```

Comment:
I don't understand this PR, environ.get() gets if exists, or falls back to the other value, which does exactly what you are trying to do now, no?

Reply:
The value is later overridden by the config (`STATIC_ROOT = section.get("static_root", STATIC_ROOT)`).